src/SIL.LCModel.Core/SpellChecking/SpellEngine.cs

```diff
@@ -168,20 +211,35 @@ private void SetInternalStatus(string word, bool isCorrect)
 			{
 				if (IsVernacular)
 				{
+					#if __MonoCS__
```

Comment:
I think this `#if` block belongs after the comment // Custom vernac....other-case matches